helm/reana/README.md

```diff
@@ -17,6 +17,7 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `components.reana_server.imagePullPolicy`                | REANA-Server image pull policy                                                       | IfNotPresent                                    |
 | `components.reana_server.uwsgi.processes`                | Number of uWSGI processes                                                            | 6                                               |
 | `components.reana_server.uwsgi.threads`                  | Number of uWSGI threads                                                              | 4                                               |
+| `components.reana_server.workflow_scheduling_policy`     | Define workflow scheduling strategy. Options are "fifo" for first-in-first-out strategy regardless of users and "balanced" for multi-user-aware scheduling strategy. | "balanced" |
```

Comment:
As this is translated directly into a config env var, what about just setting it as part of `components.reana_server.environment.REANA_WORKFLOW_SCHEDULING_POLICY`? That way we don't need changes in the reana-server template.

Reply:
(3) Also, on a similar note, due to our long variable names such as `components.reana_workflow_controller.environment.REANA_RUNTIME_KUBERNETES_KEEP_ALIVE_JOBS_WITH_STATUSES`, the markdown page is hard to read... too much horizontal scrolling necessary. Can we improve the presentation? https://github.com/reanahub/reana/blob/master/helm/reana/README.md

Reply:
(2) Yeah, I forgot to add them, my bad. If you could add them to this PR, please.

(3) I also noticed that `kubernetes_memory_limit` is missing a separator
![image](https://user-images.githubusercontent.com/8863238/122243182-34d1ab00-cec4-11eb-870c-7f6227ced3bc.png)